azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -97,8 +98,8 @@ def __impl():
                 cgroups = self._cgroups_api.create_agent_cgroups()
 
                 if track_cgroups:
-                    # TODO: Add to tracking list
-                    pass
+                    for cgroup in cgroups:
+                        CGroupsTelemetry.track_cgroup(cgroup)
```

Comment:
CGroupApi.create_agent_cgroups needs to be updated to return CGroups instead of directory paths